ui/toolbar.ini

```diff
@@ -17,7 +17,7 @@
 # Color: COLOR(0xffffff),COLOR(0xffff00),COLOR(0xff8000),COLOR(0xff00ff),COLOR(0x00ff00),COLOR(0x00c0ff),COLOR(0x808080),COLOR(0x008000),COLOR(0xff0000),COLOR(0x3333cc),COLOR(0x000000),COLOR_SELECT
 #  Notice: This are the default Xournal colors, each other color in HEX can also be used, eg COLOR(0x12ABCF);
 #
-# Tools: ERASER,PEN,HILIGHTER,IMAGE,TEXT,IMAGE,SELECT,SELECT_REGION,SELECT_RECTANGLE,VERTICAL_SPACE,HAND,DEFAULT_TOOL
+# Tools: ERASER,PEN,HIGHLIGHTER,IMAGE,TEXT,IMAGE,SELECT,SELECT_REGION,SELECT_RECTANGLE,VERTICAL_SPACE,HAND,DEFAULT_TOOL
```

Comment:
This is a breaking change. See above comment to fix.

Reply:
I would propose to still fix it here but apply the [robustness principle](https://en.wikipedia.org/wiki/Robustness_principle) (see upper comment https://github.com/xournalpp/xournalpp/pull/2339#discussion_r531128098) in the parsing. Would that work, or am I missing something.

Reply:
Yes that is what I meant. Read both versions but only write the correct one.